services/QuillLMS/app/assets/stylesheets/pages/progress_reports/progress_reports_landing_page.scss

```diff
@@ -1,131 +1,75 @@
-.progress-reports-landing-page-container {
-  margin: auto;
-  max-width: 1025px;
+.progress-reports-landing-page {
+  margin-top: 32px;
 
-  .generic-minis {
-    justify-content: flex-start
+  h1 {
+    @include display-xl;
   }
 
-  @media screen and (max-width: 1020px) {
-    .generic-minis {
-      justify-content: center;
-    }
+  h2 {
+    @include display-l-semibold;
   }
 
-  a {
-    text-decoration: none;
-  }
-  .generic-mini {
-    margin: 20px;
-  }
-  .premium {
-    margin-top: 5px;
-    color: #eda41b;
-    font-size: 12px;
-    font-weight: 600;
-    text-transform: uppercase;
-    /* want shorter images if we have the premium row of type*/
-    & + div {
-      height: 120px;
-    }
-  }
-  h3 + div {
-    padding-top: 10px;
-  }
-  .image-wrapper {
-    margin: 50 0 40 0px;
-    display: -webkit-box;
-    display: -ms-flexbox;
-    display: flex;
-    -webkit-box-align: center;
-        -ms-flex-align: center;
-            align-items: center;
-    -webkit-box-pack: center;
-        -ms-flex-pack: center;
-            justify-content: center;
-    height: 208px;
-    #responses {
-      margin-left: -75px;
-    }
+  h3 {
+    @include display-l;
   }
 
-}
+  .report-section {
+    margin-top: 32px;
 
-.progress-reports-landing-page {
-  a {
-    text-decoration: none;
-  }
-  .header {
-    display: flex;
-    justify-content: center;
-    margin: 0px 20px;
-    align-items: center;
-    text-align: left;
-    .dropdown-container {
-      min-height: 65px;
-      min-width: 200px;
-    }
-    .dropdown__control {
-      background-color: transparent;
+    > div {
```

Comment:
Optional nit: more sustainable to do a class selector here, rather than plain `div`